_source/_posts/2022-08-08-react-redux.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: blog_post
+title: Using Redux with React
+author: gabi-dombrowski
+by: contractor
+communities: [javascript]
+description: ""
+tags: [javascript, typescript, react, redux, usecontext, usestate, authentication]
+tweets:
+  - ""
+  - ""
+  - ""
+image:
+type: conversion
+canonical: https://auth0.com/blog/using-redux-with-react/
+---
+
+There are a myriad of state management options available for React. React provides the option of using the built-in [Context](https://reactjs.org/docs/context.html) for when you have a nested tree of components that share a state. There is also a built-in [useState hook](https://reactjs.org/docs/context.html) that will allow you to set local state for a component.
+
+For more complex scenarios where you might need a single source of truth that shared across large sections of your application and is frequently changing, you might want to consider using a more robust state management library.
```

Comment:
I was hoping we'd use the state management for authentication/user profile info. Will this tut cover those scenarios?

Reply:
Yes! That's the part that's missing that I already have in my demo app but need to write out in the tutorial.